studio/schemas/documents/event.ts

```diff
@@ -37,6 +38,11 @@ export default {
   title: 'Event',
   name: 'event',
   i18n,
+  validation: (Rule:Rule) => Rule.custom((fields:any) => {
+    if (fields.iframe!= null ) return "IFrame is deprecated. Please insert iframe inside the content field."
```

Comment:
The formatting is wrong. If you have prettier setup it should auto-format on save. Do you have it installed?